server/schema/tags/createTag.js

```diff
@@ -0,0 +1,26 @@
+const schema = {
+  required: ['name', 'description'],
```

Comment:
Let us make `description` to allow `NULL` values, only tag `name` would be required I guess